roles/Full App Permissions-21d7a7c6-9d68-438f-b732-e8950c5f745b.json

```diff
@@ -5,29 +5,35 @@
     "description": "Essentially the root user, use carefully",
     "name": "Full App Permissions",
     "modulePermissions": {
+        "38": {
+            "@type": "ModulePermission",
+            "canCreate": true,
+            "canExecute": true,
+            "canRead": true,
+            "canUpdate": true,
+            "canDelete": true,
+            "fieldPermissions": [],
+            "module": "keys"
+        },
         "39": {
-            "@id": "\/api\/3\/module_permissions\/ab9a5604-3602-4426-8878-5f1a8f1e033f",
```

Comment:
IRI and UUID is removed .. please check

Reply:
I have confirmed this with Backend Team. This is expected when you take role export from 7.4.0 FSR.